rte/hack/deploy-devices.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
```

Comment:
Do we want to failed hard here in case one of the created objects already exists?